tests/test_data_paths.py

```diff
@@ -21,6 +23,7 @@ def test_gdal_data_find_file():
     GDALDataFinder().find_file("header.dxf")
 
 
+@requires_gdal_lt_3
```

Comment:
In this case, we'll need to change the implementation for GDAL 3, and look for the new sqlite file in the wheel.

Also, setting PROJ_LIB is out of fashion with 3.0, there's a new function to do this: https://github.com/OSGeo/gdal/issues/1191.